### PR TITLE
frontend: make locize-pull

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

**Comment by thisconnect** [comment]
@benma thanks, please push/pull loczie keys again, as https://github.com/BitBoxSwiss/bitbox-wallet-app/pull/2992 was just merged with new keys.

**Comment by benma** [comment]
@thisconnect updated now

File: frontends/web/src/locales/de/app.json [hunk 1]
```diff
@@ -402,6 +402,55 @@
...
+              "text": "Für jeden Handel ist eine Verifizierung deiner Identität erforderlich. Dein Konto-Manager wird dich durch den Prozess begleiten und deine Fragen beantworten.",
+              "title": "Identitätsverifizierung"
+            },
+            "partnership": {
+              "text": "In Zusammenarbeit mit BTC Direct, einem vertrauenswürdigen europäischen Broker, bietet BitBox großangelegten außerbörslichen Handel (OTC) für Bitcoin, Ethereum, Stablecoins und mehr mit fachkundiger Unterstützung direkt in der BitBoxApp an.",
+              "title": "Kaufe und verkaufe über den Private Trading Desk von BTC Direct."
+            },
+            "paymentMethods": {
+              "buy": "Bitcoin kaufen:",
+              "buy2": "Gib große Kaufaufträge bei deinem persönlicher Kundenberater auf, sichere dir einen festen Preis und bezahle per Banküberweisung. Deine Kryptowährung wird nach der Zahlung direkt an deine BitBox geliefert.",
```
Comment (thisconnect) [inline frontends/web/src/locales/de/app.json:422]: persönliche**N**

File: frontends/web/src/locales/de/app.json [hunk 2]
```diff
@@ -402,6 +402,55 @@
...
+            "intro_btconly": "Erhalte persönliche Unterstützung für große außerbörsliche (OTC) Bitcoin-Handelsgeschäfte.",
+            "intro_multi": "Erhalte persönliche Unterstützung für große außerbörsliche (OTC) Handelsgeschäfte mit Bitcoin, Ethereum, Stablecoins und mehr.",
+            "kyc": "BTC Direct benötigt eine Identitätsverifizierung für alle Handelsgeschäfte, wobei dich dein persönlicher Kundenberater bei jedem Schritt unterstützt.",
+            "learnmore": "Erfahre mehr über BTC Direct's Private Trading Desk.",
+            "listItem1": "Kaufe und verkaufe jederzeit, indem du deinen persönlichen Kundenberater per Telefon, E-Mail, Videoanruf oder Chat kontaktierst.",
+            "listItem2": "Zahlungen werden nach der Auftragserteilung per Banküberweisung abgeschlossen.",
+            "listItem3": "Mindestauftragsbetrag: 50.000 €.",
+            "listItem4": "Transparente Gebühren festgelegt auf 0,5 % - 1 %, inklusive persönlicher Beratung.",
+            "manager": "Du hast einen persönlichen Kundenberater, der dich bei jedem Schritt unterstützt und dir eine sichere, unkompliziertes Abwicklung mit festen Preisen und minimaler Slippage bietet."
```
Comment (thisconnect) [inline frontends/web/src/locales/de/app.json:449]: either "sicheres, unkompliziertes Abwicklen"
or "sichere, unkomplizierte~~S~~ Abwicklung"

**Review by thisconnect — CHANGES_REQUESTED** [review]

**Review by thisconnect — APPROVED** [review]
LGTM with missing ES translation.

**Pull request merged**